account_payment_partner/views/account_move_view.xml

```diff
@@ -23,22 +23,18 @@
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
         <field name="arch" type="xml">
-            <xpath
-                expr="//div[field[@name='invoice_payment_term_id']]"
-                position="after"
-            >
+            <field name="partner_bank_id" position="before">
```

Comment:
You are moving this to a place that is not always shown (I don't see it in customer invoices) and not in main section. Please put it before `journal_id` instead.